CI/travis.validate_deployment.sh

```diff
@@ -2,7 +2,7 @@
 
 if [ -z "${TRAVIS_TAG}" ]; then
   echo "Not a release build - skipping release validation."
-  exit 0
+  exit
```

Comment:
Is this (or the previous paragraph) in the Travis documentation relevant? https://docs.travis-ci.com/user/job-lifecycle#breaking-the-build

Reply:
I realise now that it is the previous paragraph that is relevant, it is quitting because of the `exit` call. Fixing now...